tool/tctl/common/resource_command.go

```diff
@@ -654,9 +654,18 @@ func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI,
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	if err := server.CheckAndSetDefaults(); err != nil {
+		return trace.Wrap(err)
+	}
 
-	if !rc.force {
-		return trace.AlreadyExists("nodes cannot be created, only upserted")
+	name := server.GetName()
+	_, err = client.GetNode(ctx, server.GetNamespace(), name)
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	exists := (err == nil)
+	if !rc.IsForced() && exists {
+		return trace.AlreadyExists("node %q already exists, use -f flag to override", name)
```

Comment:
```suggestion
		return trace.AlreadyExists("node %q already exists, use --force flag to override", name)
```

In general, prefer the long flags names in writing, help messages, docs, etc.

The short aliases are nice for interactive use, but aren't descriptive enough for docs and error messages.
```suggestion
		return trace.AlreadyExists("node %q already exists, use -f flag to override", name)
```

Reply:
Will the `name` always be a random UUID? If so, including address and hostname in the message would be helpful as well.